contracts/liquidityMining/KyberFairLaunch.sol

```diff
@@ -42,30 +45,34 @@ contract KyberFairLaunch is IKyberFairLaunch, PermissionAdmin, ReentrancyGuard {
     //   4. User's `amount` gets updated.
   }
 
+  struct PoolRewardData {
+    uint256 rewardPerBlock;
+    uint256 accRewardPerShare;
+  }
   // Info of each pool
-  // rewardPerBlock: amount of reward token per block
-  // accRewardPerShare: accumulated reward per share of token
+  // poolRewardData: reward data for each reward token
+  //      rewardPerBlock: amount of reward token per block
+  //      accRewardPerShare: accumulated reward per share of token
   // totalStake: total amount of stakeToken has been staked
   // stakeToken: token to stake, should be an ERC20 token
   // startBlock: the block that the reward starts
   // endBlock: the block that the reward ends
+  // rewardTokens: list of reward tokens for this pool
   // lastRewardBlock: last block number that rewards distribution occurs
   struct PoolInfo {
-    uint128 rewardPerBlock;
-    uint128 accRewardPerShare;
     uint256 totalStake;
     address stakeToken;
     uint32 startBlock;
     uint32 endBlock;
     uint32 lastRewardBlock;
+    address[] rewardTokens;
```

Comment:
pls add a comment that rewardTokens and stakeToken are immutable

Reply:
actually I added `addRewardToken` function to allow add new reward tokens, for removing reward tokens, we can just set the rewardPerBlock = 0